packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -424,6 +424,42 @@ describe("AuthCodeRedirectHandler", () => {
     ).toEqual({});
   });
 
+  it("swallows the exception if the fetch to the session endpoint fails", async () => {
+    window.fetch = jest
+      .fn()
+      .mockReturnValueOnce(
+        new Promise((resolve) => {
+          resolve(
+            new Response("https://my.webid", {
+              status: 200,
+            })
+          );
+        })
+      )
+      .mockImplementation(() => {
+        throw new Error("Some error");
+      });
```

Comment:
This can be simplified with `mockResolvedValue`. Additionally, it's a rejected Promise (since it's an async function), not an exception, so I suppose that would more accurately use `mockRejectedValue`?

```suggestion
      .mockResolvedValueOnce(
        new Response("https://my.webid", {
          status: 200,
        })
      )
      .mockRejectedValueOnce(new Error("Some error"));
```

Reply:
Right, that looks much better: c2b1f5bf971b5b781bc45d64550b5326336de6c4

Reply:
Actually let's fix the root cause ^^ c0b777bf192af90006e1acb3a1397df28b7b2e5c